### PR TITLE
Add a ToS design change mention in the changelog, close #6121

### LINKED ISSUE #6121: Terms of service design broken
The port to bootstrap 3 broke the ToS page which now uses the full width. It only misses a lg8 somewhere.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/22451706-terms-of-service-design-broken?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Flaburgan**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ changed. diaspora\* will no longer listen on `0.0.0.0:3000` as it will now
 bind to an UNIX socket at `unix:tmp/diaspora.sock`. Please change your local
 `diaspora.yml` if necessary.
 
+## Terms of Use design changes
+
+With the port to Bootstrap 3, the app/views/terms/default.haml file changed. If you edited the terms by adding a terms.erb file, you probably want to backup these changes in it
```
Comment (svbergerem) [inline Changelog.md]: In https://github.com/diaspora/diaspora/blob/develop/config/diaspora.yml.example#L496 we mention `app/views/terms/terms.haml` and `app/views/terms/terms.erb` so we should also mention both here.

I'm also not sure about the wording "backup these changes in it". `app/views/terms/default.haml` now has a new structure and the podmin should make sure that his/hers terms file has the same structure. Maybe @goobertron could help us here.

**Comment by goobertron** [comment]
OK, not sure if I'm understanding the issues, but from what I gather, how about:

> With the port to Bootstrap 3, app/views/terms/default.haml has a new structure. If you have created a customised app/views/terms/terms.haml or app/views/terms/terms.erb file, you will need to edit the new app/views/terms/default.haml file to include your customisations, and save this new file as app/views/terms/terms.haml or app/views/terms/terms.erb.

It sounds as though existing app/views/terms/terms.haml or app/views/terms/terms.erb files will no longer be compatible, so podmins will need to include their customised text in the new structure (provided in app/views/terms/default.haml) and save over the top of their old customised app/views/terms/terms.haml or app/views/terms/terms.erb file. I hope I've described it correctly - let me know of any misunderstandings or lack of clarity in my draft and I'll attempt to improve it.

**Comment by svbergerem** [comment]
@goobertron lgtm

**Comment by svbergerem** [comment]
@denschub: @Flaburgan didn't apply the changes @goobertron suggested. Could you change the Changelog or should someone else do that?

**Comment by Flaburgan** [comment]
Oh sorry, I was in vacation... I'll open a new PR.

**Comment by svbergerem** [comment]
@Flaburgan No problem, not your fault. I guess @denschub just misunderstood my [last comment](https://github.com/diaspora/diaspora/pull/6122#issuecomment-118412845).

**Comment by denschub** [comment]
Too... many... open... PRs. Must! close! them! all!

**Pull request merged**